devfile.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: 1.0.0
+metadata:
+  generateName: che-theia-
+
+components:
+  - mountSources: true
+    command:
+      - tail
+      - '-f'
+      - /dev/null
+    memoryLimit: 512Gi
+    type: dockerimage
+    image: 'quay.io/eclipse/che-theia:next'
+    alias: che-theia-next-dev
+    env:
+      - value: 0.0.0.0
+        name: THEIA_HOST
+      - value: '3130'
+        name: THEIA_PORT
+      - value: '0'
+        name: NODE_TLS_REJECT_UNAUTHORIZED
+
+  - mountSources: true
+    memoryLimit: 3Gi
+    type: dockerimage
+    image: 'quay.io/eclipse/che-theia-dev:next'
+    alias: che-dev
```

Comment:
minor: I found the name of containers quite strange because container with image of theia is called che-theia-next-dev and this one is called che-dev

Reply:
yeah this is a problem of dog fooding ... che-theia everywhere ... : What about:
- `che-theia-next-dev` -> `che-theia-next-image`
- `che-dev` -> `dev`

Reply:
prod/dev :-)

Reply:
`che-theia-next-prod` and `dev` would be ok ?

Reply:
event short name `che-theia` and `dev` or `che-theia-runtime` 
but I'm fine with your or other proposals

Reply:
ok it will be
`che-theia-next-runtime` and `dev`, to me it has to clear that it is the prod/runtime container.